src/gui/rss/rsswidget.h

```diff
@@ -91,6 +91,7 @@ private slots:
 
 private:
     bool eventFilter(QObject *obj, QEvent *event) override;
+    void convertRelativePathToAbsolute(QString &html, const QString &basePath) const;
```

Comment:
This function does not need (and isn't supposed to need) access to `RSSWidget` class members, so it is preferable to declare it in anonymous namespace in `rsswidget.cpp` file.